wagtail_localize/views/edit_translation.py

```diff
@@ -1078,22 +1130,23 @@ def user_can_edit_instance(user, instance):
         return user_can_edit_snippet_type(user, instance.__class__)
 
 
+@method_decorator(xframe_options_sameorigin_override)
 def preview_translation(request, translation_id, mode=None):
     translation = get_object_or_404(Translation, id=translation_id)
 
     instance = translation.get_target_instance()
 
-    if not isinstance(instance, Page):
-        raise Http404
+    # if not isinstance(instance, Page):
+    #     raise Http404
```

Comment:
question: why are these commented out?

if they are not applicable in 4.0, then we want to wrap them in a `if WAGTAIL_VERSION >= (4, 0, 0)`

If more, then they need removing (and ideally the commit should have a note about why in the description)

Reply:
Ah sorry, this was a mistake, will revert